sklearn/metrics/tests/test_ranking.py

```diff
@@ -1468,3 +1469,54 @@ def test_partial_roc_auc_score():
         assert_almost_equal(
             roc_auc_score(y_true, y_pred, max_fpr=max_fpr),
             _partial_roc_auc_score(y_true, y_pred, max_fpr))
+
+
+@pytest.mark.parametrize('k, expected_score', [(2, .2), (3, .4), (4, .8)])
```

Comment:
let's also check the `normalize=Flase` case?

Reply:
It's checked in the docstring with the `y_true = [0]*4` case. Do you want me to move them into `test_ranking.py`?

Reply:
Added both to `test_ranking.py`